bin/git/git.js

```diff
@@ -8,13 +8,13 @@ const debug = setupDebug('chromatic-cli:git');
 
 async function execGitCommand(command) {
   try {
-    return execSync(`${command} 2>&1`)
-      .toString()
-      .trim();
+    const { stderr, stdout } = await execa.command(command, {
+      env: { LANG: 'C', LC_ALL: 'C' }, // make sure we're speaking English
+      timeout: 10000, // 10 seconds
+    });
+    return stderr || stdout;
```

Comment:
I'm not sure this is the same behaviour (`2>&1` interleaves stderr & stdout) but it is probably OK for our purposes.

Reply:
Still could we use this? https://github.com/sindresorhus/execa#all

Reply:
👍